### DISCUSSION TITLE
Commercial playlist

### DISCUSSION THREAD

**Discussion by radiobellissima** [AzuraCast/AzuraCast#3997] [Q&A]
I have tried in every way to have a sequential playlist made up of 3 files every 20 minutes or scheduled by time on the air. Is it possible that there is no way to do this?

**Comment by Jaadoo13** [comment]
Hello. If you make a new Playlist, then you have in the first Tab the "Playlist Type". There you can choose "play once each x minutes.." check to activate this list and save it. this works for me. As i use it for my Radio Jingles to be played every 25 Minutes.

**Comment by radiobellissima** [comment]
Sure it works. But I have to play an entire playlist not a single file. This doesn't work.

**Comment by Jaadoo13** [comment]
Okay, i tested it again after  your message. And youre right. Actually it plays only the first song of this playlist. I dont know why.
But you can make 5 playlists and for the first every 10minutes, for the second every 11minutes... Maybe it helps as long there is no fix. 

Thats really bad thing. I hope also that it will be possible, to set the entire playlist to be played every x minutes (for ads, because they short enough)

Edit: I testet all possible Settings, now. But the only thing you can make, is to play every x (Song or Minute). And it plays then on every loop the next Song from the playlist. but ya, not all in one loop. :(

**Comment by radiobellissima** [comment]
Unfortunately I can not do it because the files are relatively short are the accesses to the advert and last a few seconds. In any case, even testing with longer files, everything is then not respected as an order so it is practically unusable.

**Comment by Jaadoo13** [comment]
Okay, what if you make one big File of all the short Files? Then you have only one File and this works. I mean as long there is no change of possibilities in azuracast....

**Comment by radiobellissima** [comment]
I can't do that, the commands are serving the separate tags.

**Comment by silvaalex2021** [comment]
@radiobellissima you can (almost) get what you want using the "merge playlist to play as a single track" option, which is present in the "advanced" tab of the playlist settings dialog.

although this is the only way, it will probably break the schedule, because azuracast's internal autoDJ will continue to feed Liquidsoap with new tracks, unaware that there is a programmed playlist on the way.

summary: your commercials will play, but only when liquidsoap finds an opportunity to do so, which can take a long time.

P.S .: I have been testing this for months with a replica of my station, and this is exactly the behavior I encountered in practice. my choices are to deactivate Azuracast's internal AutoDJ and rely only on LS, or accept that only one commercial will be played every X minutes...

**Comment by radiobellissima** [comment]
This solution is not practicable. And I also believe it is an indispensable function for any broadcaster. Leaving aside the commercial side but not being able to transmit a playlist at a certain time or date makes the software not very usable for any radio.

**Comment by silvaalex2021** [comment]
Please don't take it that way. There are a lot of good people trying to make azuracast as useful as possible for us, broadcasters and service providers. By calling their software "unusable", we offend their work and demotivate them instead of helping.

Regarding the solution that I presented, I tested it for a long time and yes, it is perfectly possible to be applied. Of course, by disabling azuracast's internal autoDJ, you will lose some functionality, but if that's what works for you, it's better to use what we have and wait for our friends to implement some things that are still missing from azuracast's internal autoDJ.

**Comment by radiobellissima** [comment]
Perhaps my concept is misperceived by using the translator. It wasn't an offense to the developers, I've been using it for years on all my channels and I'm delighted with the work done. As for the playlists, I can't send them on the air randomly, you need the exact date and time so for the moment I continue with live broadcasts with my direction. I am sure it will be fixed soon and so I can free some of my live servers.

**Comment by silvaalex2021** [comment]
you can help by voting on a suggestion I made on the azuracast featurebot, here: https://features.azuracast.com/suggestions/159484/add-an-option-to-play-the-entire-playlist-once-per-x-songs-or-once-per-x-minutes
